metrics_ci_integration/lib/jenkins/client/model/jenkins_build_artifact.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:ci_integration/jenkins/client/model/jenkins_build.dart';
+import 'package:equatable/equatable.dart';
+
+/// A class representing a single artifact of a [JenkinsBuild].
+class JenkinsBuildArtifact extends Equatable {
+  /// A name of this artifact file.
+  final String fileName;
+
+  /// A path to this artifact.
```

Comment:
Maybe we should mention that this path points to the artifact that on the Jenkins instance?